frontend/apps/cloudstorage/webdav.lua

```diff
@@ -50,7 +50,7 @@ The start folder is appended to the server path.]])
 
     local hint_name = _("Server display name")
     local text_name = ""
-    local hint_address = _("WebDAV address eg https://example.com/dav")
+    local hint_address = _("WebDAV address e.g. https://example.com/dav")
```

Comment:
Formally speaking:
```suggestion
    local hint_address = _("WebDAV address (e.g., https://example.com/dav)")
```

But maybe just:
```suggestion
    local hint_address = _("WebDAV address, for example https://example.com/dav")
```

Reply:
I do like the second string.

Reply:
It'll probably go easier on non-Western translators, for which Latin may not be as pervasive in their native language/culture ;).